postcss.config.js

```diff
@@ -16,22 +15,25 @@ const unrgba = require('postcss-unrgba')
  * @param {string | import('vinyl')} [context.file] - File path or object
  * @returns {import('postcss-load-config').Config} PostCSS config
  */
-module.exports = ({ env, file = '' }) => {
-  const { dir, name } = parse(typeof file === 'object'
-    ? file.path // Normalise to string path
-    : file
-  )
+module.exports = ({ env, file }) => {
+  const plugins = []
+
+  // Normalise to file object
+  const { basename, dirname } = typeof file === 'string'
+    ? new Vinyl({ path: file })
```

Comment:
We do this because:

1. PostCSS CLI uses [`{ dirname, basename, extname }`](https://github.com/postcss/postcss-cli#context)
2. Plugin [`gulp-postcss`](https://github.com/postcss/gulp-postcss#readme) uses Vinyl file objects
3. Plugin [`postcss-loader`](https://github.com/webpack-contrib/postcss-loader#readme) uses path strings

We can likely review this as this issues continues:

* https://github.com/alphagov/govuk-frontend/issues/2717

Reply:
I've moved these changes to https://github.com/alphagov/govuk-frontend/pull/3259 as they blocked the `gulp-postcss` removal